acceptance-tests/src/test/resources/ibft/ibft.json

```diff
@@ -0,0 +1,43 @@
+{
+  "config": {
+    "chainId": 4,
+    "homesteadBlock": 1,
+    "eip150Block": 2,
+    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "eip155Block": 3,
+    "eip158Block": 3,
+    "byzantiumBlock": 1035301,
+    "revisedibft": {
+      "blockperiodseconds": 5,
```

Comment:
do we want to drop these values to make tests faster? (blocktime could be 1 reasonably).

If we do timeout tests, we probably don't want to wait 25 seconds for (request) timeout ....

Reply:
not sure what reasonable time is. in the clique tests I found they were not as reliable when I had the block time at such a small value. will have to think about this a bit. but in general want this to be as small as practical.
agree 25 seconds is too long. ideas on how long? I might go with 5s for now.

Reply:
done